### ISSUE TITLE
Multi-cursor support

### ISSUE THREAD

**Issue by baileygriff** [kaiwood/vscode-endwise#17]
Are there any plans to support multiple cursors in the future?

**Comment by kaiwood** [comment]
Absolutely, as I'm a big proponent of multi-cursors. Should be the next improvement after support for more languages (already working on that, should be ready really soon).

**Comment by kaiwood** [comment]
https://github.com/kaiwood/vscode-endwise/commit/68b0f87cf5e281edcf1e3c0449ac888a6c1ab5bf
